### PR TITLE
Feature/Enable host configuration for dev wallet

### LINKED ISSUE #515: Enable host configuration for dev wallet
The dev-wallet is run and uses a default host for access node connection, this host should be passed as an option using a command flag. 
The default value is set here https://github.com/onflow/flow-cli/blob/master/internal/tools/wallet.go#L68
It should be exposed using a flag.

### PR DESCRIPTION
**Pull request by sukantoraymond**
Closes #515

## Description

This PR enable cli users to add custom configuration for dev-wallet host by adding the flag --wallet-host in dev-wallet command. Example of this command is flow dev-wallet --wallet-host "http://localhost:8081" (default value is http://localhost:8080)
______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: internal/tools/wallet.go [hunk 1]
```diff
@@ -32,7 +32,8 @@ import (
 )
 
 type FlagsWallet struct {
-	Port uint `default:"8701" flag:"port" info:"Dev wallet port to listen on"`
+	Port uint   `default:"8701" flag:"port" info:"Dev wallet port to listen on"`
+	Host string `default:"http://localhost:8080" flag:"wallet-host" info:"Host for access node connection"`
```
Comment (sideninja) [inline internal/tools/wallet.go:36]:
```suggestion
	Host string `default:"http://localhost:8080" flag:"emulator-host" info:"Host for access node connection"`
```

**Review by sideninja — CHANGES_REQUESTED** [review]
There are no doc updates.

File: internal/tools/wallet.go (hunk 1 above)
Comment (sukantoraymond) [reply internal/tools/wallet.go]: addressed

File: internal/tools/wallet.go [hunk 2]
```diff
@@ -32,7 +32,8 @@ import (
 )
 
 type FlagsWallet struct {
-	Port uint `default:"8701" flag:"port" info:"Dev wallet port to listen on"`
+	Port uint   `default:"8701" flag:"port" info:"Dev wallet port to listen on"`
+	Host string `default:"http://localhost:8080" flag:"emulator-host" info:"Host for access node connection"`
```
Comment (psiemens) [inline internal/tools/wallet.go:36]:
```suggestion
	Host string `default:"http://localhost:8888" flag:"emulator-host" info:"Host for access node connection"`
```

It looks like the emulator bundled with the CLI no longer exposes a server on port 8080. Should this be `http://localhost:8888` now?

File: docs/tools.md [hunk 3]
```diff
@@ -47,14 +47,15 @@ You can read more about setting up dev-wallet at [FCL Dev Wallet Project](https:
...
 - Default: `8701`
 
-Port on which the dev wallet server will listen on. 
+Port on which the dev wallet server will listen on.
 
-### Emulator Port
+### Emulator Host
 
-- Flag: `--emulator-port`
-- Valid inputs: Number
```
Comment (sukantoraymond) [inline docs/tools.md:55]: deleted this because its not on the flags list in codebase

**Review by sideninja — APPROVED** [review]
Thank you, LGTM

**Pull request merged**